include/zephyr/fs/zms.h

```diff
@@ -82,125 +80,126 @@ struct zms_fs {
  */
```

Comment:
It's one API that we're talking about, not several. (On the `@defgroup` line above.)